web/packages/teleport/src/types.ts

```diff
@@ -29,13 +29,59 @@ export interface Context {
 }
 
 export interface TeleportFeatureNavigationItem {
-  title: string;
+  title: NavTitles;
   icon: React.ReactNode;
   exact?: boolean;
   getLink?(clusterId: string): string;
   isExternalLink?: boolean;
 }
 
+export type NavTitle = `${NavTitles}`;
```

Comment:
What's this used for?

Reply:
Thanks for pointing out, I defined it to be used as a type built from enum but ended up not using it anywhere. Removed in https://github.com/gravitational/teleport/commit/007fcdb19d5d45a2e664b0aa9208dbe9b446e9ee